src/test/java/service/ReadFileServiceImplTest.java

```diff
@@ -0,0 +1,33 @@
+package service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReadFileServiceImplTest {
+    private static ReadFileService readFileService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        readFileService = new ReadFileServiceImpl();
+    }
+
+    @Test
+    public void read_fileWithCorrectInput_ok() {
```

Comment:
```suggestion
    public void read_fileWithCorrectFilePath_ok() {
```
It's better to specify what kind of input mehod achive.
For example `emptyList`, `ivalidPath` or `nullObject`